src/components/planning/LabwarePlan.tsx

```diff
@@ -386,19 +388,29 @@ function buildValidationSchema(labwareType: LabwareType): Yup.AnyObjectSchema {
     labwareType.name === LabwareTypeName.VISIUM_TO ||
     labwareType.name === LabwareTypeName.VISIUM_ADH ||
     labwareType.name === LabwareTypeName.XENIUM
+  ) {
+    formShape.costing = Yup.string().oneOf(Object.values(SlideCosting)).required('Slide costing is a required field');
+  }
+  if (
+    labwareType.name === LabwareTypeName.VISIUM_LP ||
+    labwareType.name === LabwareTypeName.VISIUM_TO ||
+    labwareType.name === LabwareTypeName.VISIUM_ADH
   ) {
     formShape.lotNumber = Yup.string()
       .required()
       .matches(/^\d{6,7}$/, 'Slide lot number should be a 6-7 digits number');
-    formShape.costing = Yup.string().oneOf(Object.values(SlideCosting)).required('Slide costing is a required field');
+  } else if (labwareType.name === LabwareTypeName.XENIUM) {
+    formShape.lotNumber = Yup.string()
+      .required()
+      .matches(/^\d-\d{4}[A-Za-z]$/, 'Slide lot number should be in format: Digit, hyphen, 4 digits, letter');
```

Comment:
To my understanding it is only capital letter at the last part. Please confirm with David

Reply:
I am looking into the core codebase, and I think the validation is been missed there, I will check with David